configuration_test.py

```diff
@@ -16,28 +16,29 @@ def configuration_test(ARGUMENTS):
         sys.tracebacklimit = 0 
 
     # Checks initial prerequisites
-    configuration_tests.check_python(gslab_python_version = '3.0.5', 
-                                     packages = ["yaml", "gslab_scons", "gslab_fill"])
-    configuration_tests.check_lyx()
-    configuration_tests.check_lfs()
+    config.check_python(gslab_python_version = gslab_python_version, 
+                        packages = ["yaml", "gslab_scons", "gslab_fill"])
+    config.check_lyx()
+    config.check_lfs()
 
     # Uncomment if using
-    # configuration_tests.check_r(packages = ["yaml"]) 
+    # config.check_r(packages = ["yaml"]) 
 
     # Loads arguments and configurations
     mode = ARGUMENTS.get('mode', 'develop') # Gets mode; defaults to 'develop'
 
     # Check stata
-    sf = configuration_tests.check_stata(["yaml"])
+    sf = config.check_stata(["yaml"])
```

Comment:
Why is the stat check separate from the default checks and below the optional R check? I'd think we'd want it with the defaults or at least in a prominent position.